src/platform/landing-pages/header-footer-data.json

```diff
@@ -494,7 +494,7 @@
               "title": "Get employment benefits",
               "links": [
                 {
-                  "text": "About Vocational Rehabilitation and Employment",
+                  "text": "About Veteran Readiness and Employment",
```

Comment:
The page that both of these links go to has an `<h1>` tag with Veteran Readiness and Employment as its content. Changing these align with that title. It might be worth while to update the href at some point down the line, but that's outside the scope of this ticket.